x-pack/plugin/sql/qa/src/main/resources/alias.csv-spec

```diff
@@ -28,13 +28,13 @@ DESCRIBE test_alias;
 
        column       |     type      |    mapping    
 --------------------+---------------+---------------
-birth_date          |TIMESTAMP      |DATE           
+birth_date          |TIMESTAMP      |DATETIME
```

Comment:
Ideally we should change `mapping` to be lower-case to follow the ES convention - this can be done in a follow-up PR.
```
birth_date | TIMESTAMP | datetime
dep        | STRUCT    | nested
```